### ISSUE TITLE
getGeneratedKeys for PreparedStatement

### ISSUE THREAD

**Issue by migel** [SQLDroid/SQLDroid#50]
Hi,

The ```getGeneratedKeys``` method of  ```SQLDroidPreparedStatement``` throws a "not implemented" exception but the same method in ```SQLDroidStatement``` is implemented as:

```java
@Override
public ResultSet getGeneratedKeys() throws SQLException {
    execute("SELECT last_insert_rowid()");
    return rs;
}
```

Is there a reason that it can't be implemented the same in ```SQLDroidPreparedStatement```? I've tried it and it seems to work; but I have very little experience with JDBC drivers; is there something I'm missing about it?

**Comment by dperiwal** [comment]
Fixed with pull request #65. Thanks,

**Comment by dperiwal** [comment]
Already closed with pull request #65

**Comment by donv** [comment]
Yeah, just added it to the release milestone.  Thinking of making a release Real Soon Now™️.
